bin/kafka-rest-run-class

```diff
@@ -39,6 +39,11 @@ if [ "x$KAFKAREST_LOG4J_OPTS" = "x" ]; then
   fi
 fi
 
+# Log directory to use
+if [ "x$LOG_DIR" = "x" ]; then
```

Comment:
[[ -z "${LOG_DIR+x}" ]] && LOG_DIR="$base_dir/logs"

Reply:
BTW, not sure if `$base_dir/logs` is a better default log folder than, for example, `/var/log/kafka-rest`. AK uses `/var/log/kafka` by default, I think.